api/poi.go

```diff
@@ -37,7 +40,9 @@ func (s *Server) addPOI(c *gin.Context) {
 		return
 	}
 
-	poi, err := s.mongoStore.AddPOI(account.AccountNumber, body.Alias, body.Address,
+	placeType := utils.ReadPlaceType(body.Types)
```

Comment:
Using "type PalaceType string"  is more implicit of the POI type of data.

Reply:
We originally have `SymptomType` and it ends up change back to `string`. I tend to not change this for now. And we could open a discussion when later on to review this kind of values.